src/libsyntax/feature_gate.rs

```diff
@@ -641,6 +638,8 @@ declare_features! (
     (removed, extern_in_paths, "1.33.0", Some(55600), None,
      Some("subsumed by `::foo::bar` paths")),
     (removed, quote, "1.33.0", Some(29601), None, None),
+    // Allows using `#[unsafe_destructor_blind_to_params]` (RFC 1238).
+    (removed, dropck_parametricity, "1.38.0", Some(28498), None, None),
```

Comment:
Is this correct? This features is unstable since `1.3.0`.

Reply:
AFAIK unstable features just disappear entirely, no need to keep any trace of them around.